### ISSUE TITLE
Using a numerical string also hooks the scan code 

### ISSUE THREAD

**Issue by FlamingKetchup** [boppreh/keyboard#392]
Using a numerical string i.e. `"4"` or even `"num 4"` with `add_hotkey` also hooks the scan code (In this case, left arrow). If you have hook both the number and the key with corresponding scan code, pressing the latter will also run the former.

```Python
import keyboard

keyboard.add_hotkey("4", print, args=["Four was pressed"])
keyboard.add_hotkey("left", print, args=["Left was pressed"])
keyboard.wait()
```
Output when pressing 4 (expected behavior): 
```
Four was pressed
```
Output when pressing left arrow: 
```
Four was pressed
Left was pressed
```

**Comment by FlamingKetchup** [comment]
This seems to be an issue with adding a hotkey for the numerical string. For example, removing `keyboard.add_hotkey("left", print, args=["Left was pressed"])` has the same issue of outputting `Four was pressed` when pressing left arrow.

**Comment by zizihe113-network** [comment]
Yes, I used the function keyboard.press('left') to press "left arrow".But in fact,it pressed "num 4" ranther than the arrow.
